Casks/t/there.rb

```diff
@@ -1,23 +1,25 @@
 cask "there" do
-  version "1.8.10"
-  sha256 "8c5b6e2e8a35d0f1088c15cf8d4f9b5a407feeeeccbc764a08098c354a181939"
+  version "2.0.0"
+  sha256 "5a9dd8eebbb385f6fa09309b559f4d0341231519f39527bea2315140f7301c7f"
 
-  url "https://github.com/therehq/there-desktop/releases/download/v#{version}/There-#{version}-mac.zip",
-      verified: "github.com/therehq/there-desktop/"
+  url "https://github.com/dena-sohrabi/There/releases/download/v#{version}/There.zip",
+      verified: "github.com/dena-sohrabi/There/"
   name "There"
-  desc "Tool to display the local times of people in a team"
+  desc "Tool to display the local times of friends, teammates, cities or any time zone"
   homepage "https://there.pm/"
 
   livecheck do
-    url :url
-    strategy :github_latest
+    strategy :github_releases
```

Comment:
```suggestion
    url :url
    strategy :github_latest
```

This should be restored to `GithubLatest`, as the `GithubReleases` strategy isn't necessary here. `GithubLatest` checks the "latest" release on GitHub, whereas `GithubReleases` checks multiple recent releases (so it has to transfer more data, adding to overhead). We only use `GithubReleases` when checking the "latest" release doesn't work as expected (and we don't have other options).

Besides that, all the existing `livecheck` strategies other than `ExtractPlist` require a `url` value, so it shouldn't be omitted here.

Reply:
Thanks for the review, I applied your suggestion.